include/dt-bindings/gpio/gpio.h

```diff
@@ -76,29 +76,26 @@
 
 /** @} */
 
+/* Note: Bits 15 downto 8 are reserved for SoC specific flags. */
+
 /**
- * @name GPIO pin voltage flags
- *
- * The voltage flags are a Zephyr specific extension of the standard GPIO
- * flags specified by the Linux GPIO binding. Only applicable if SoC allows
- * to configure pin voltage per individual pin.
- *
+ * @deprecated Use the GPIO controller/SoC specific `*_GPIO_VOLTAGE_*` flags instead.
  * @{
  */
 
 /** @cond INTERNAL_HIDDEN */
-#define GPIO_VOLTAGE_POS	6
-#define GPIO_VOLTAGE_MASK	(3U << GPIO_VOLTAGE_POS)
+#define GPIO_VOLTAGE_POS        11
+#define GPIO_VOLTAGE_MASK       (3U << GPIO_VOLTAGE_POS)
 /** @endcond */
 
 /** Set pin at the default voltage level */
-#define GPIO_VOLTAGE_DEFAULT	(0U << GPIO_VOLTAGE_POS)
+#define GPIO_VOLTAGE_DEFAULT    (0U << GPIO_VOLTAGE_POS)
```

Comment:
Shouldn't `__DEPRECATED_MACRO` be added also to `GPIO_VOLTAGE_*` macros?

Reply:
As far as I can tell, our tools do not support `__DEPRECATED_MACRO` in dts files.